substra/sdk/schemas.py

```diff
@@ -54,13 +54,21 @@ class Type(enum.Enum):
     ComputePlan = "compute_plan"
     Organization = "organization"
     Task = "task"
-    SummaryTask = "summary_task"
+    InputAsset = "input_asset"
+    OutputAsset = "output_asset"
 
     def to_server(self):
         """Returns the name used to identify the asset on the backend."""
         name = self.value
         return _SERVER_NAMES.get(name, name)
 
+    def to_asset_kind(self):
+        """Returns the name used to identify the asset on the backend."""
+        if self.value == self.Dataset.value:
+            return AssetKind.data_manager
+        else:
+            return AssetKind[self.value]
```

Comment:
```suggestion
```
No use

Reply:
It's actually used there: https://github.com/Substra/substra/blob/41b9a24a76fec12fa84dc2bff62122ffae352481/substra/sdk/backends/local/compute/worker.py#L322